noir-projects/noir-contracts/contracts/nft_contract/src/types/nft_note.nr

```diff
@@ -1,15 +1,14 @@
 use dep::aztec::{
-    keys::getters::{get_nsk_app, get_public_keys},
-    macros::notes::partial_note,
-    note::utils::compute_note_hash_for_nullify,
-    oracle::random::random,
+    keys::getters::{get_nsk_app, get_public_keys}, macros::notes::partial_note,
+    note::utils::compute_note_hash_for_nullify, oracle::random::random,
     prelude::{NoteHeader, NullifiableNote, PrivateContext},
     protocol_types::{
```

Comment:
The CI is complaining about this bit of code. Probably just rolling it back to how it is on master should do the trick.